homeassistant/components/insteon_plm.py

```diff
@@ -0,0 +1,96 @@
+"""
+Support for INSTEON PowerLinc Modem.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/insteon_plm/
+"""
+import logging
+import asyncio
+
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_PORT, EVENT_HOMEASSISTANT_STOP)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers import discovery
+
+REQUIREMENTS = ['insteonplm==0.7.4']
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_PORT = '/dev/ttyUSB0'
+DOMAIN = 'insteon_plm'
+
+CONF_OVERRIDE = 'device_override'
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.string,
+        vol.Optional(CONF_OVERRIDE, default=[]): vol.All(
+            cv.ensure_list_csv, vol.Length(min=1))
+    })
+}, extra=vol.ALLOW_EXTRA)
+
+PLM_PLATFORMS = [
+    'binary_sensor', 'light', 'switch'
+]
+
+
+@asyncio.coroutine
+def async_setup(hass, config):
+    """Set up our connection to the PLM."""
+    import insteonplm
+
+    conf = config[DOMAIN]
+    port = conf.get(CONF_PORT)
+    overrides = conf.get(CONF_OVERRIDE)
+
+    _LOGGER.info('Looking for PLM on %s', port)
+
+    plm = yield from insteonplm.Connection.create(
+        device=port, loop=hass.loop)
+
+    print(overrides)
```

Comment:
If you'd like this to be printed to the log, would you mind using something more like:

    _LOGGER.debug('Overrides: %s', overrides)